### ISSUE TITLE
How to build training data

### ISSUE THREAD

**Issue by giselerudderham** [zsylvester/segmenteverygrain#17]
Hello, in the python notebook there is refence to creating training data when the UNET segmentation quality is low using the 'Finetuning the base model' code at the end of the notebook. It specifies that the images and masked images should be created as training data but doesn't specify exactly what the masked images should look like/how they should be prepared. In the 'Getting started.rst' file it references a python script called Train_seg_unet_model.ipynb that goes through the steps to create training data and train the model however the link to that notebook is broken. Thanks!
